config/config_request.go

```diff
@@ -279,26 +226,6 @@ func (x *NodeRequest) WithAuditSinkConfig(v *configpb.AuditSinkConfig) *NodeRequ
 	return x
 }
 
-func (x *NodeRequest) WithTokenIntrospectConfig(v *configpb.TokenIntrospectConfig) *NodeRequest {
```

Comment:
Why it is deleted? I just added that. It is required by Terraform.

Reply:
oops! wrong rebase! Fortunately you saw this